client/extensions/wp-super-cache/miscellaneous.jsx

```diff
@@ -63,6 +63,14 @@ const Miscellaneous = ( {
 							</span>
 						</FormToggle>
 						}
+						{ ! compressionDisabledMessage &&
+						<Notice
+							isCompact
+							className="wp-super-cache__toggle-notice"
+							text={ translate( 'Compression is disabled by default because some hosts have problems ' +
+								'with compressed files. Switching it on and off clears the cache.' ) }
+							/>
+						}
```

Comment:
I don't quite get why we're displaying this here -- isn't that what [R48](https://github.com/Automattic/wp-calypso/pull/14714/files#diff-1beab3e7e29dc0d31604bc20df1c8de3R48) is supposed to do?